Example/Example/Main.storyboard

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@jsani-radar @jrtibbetts @phil-brown Cool that we're leveling up the iOS example app. The iOS and Android example apps are starting to diverge a bit though. Do we care? Worth investing in keeping them in sync?

Reply:
If we want to keep this example app headless, maybe I can add the SDK version number to the toolkit instead.

Reply:
I think this would be extremely useful in the toolkit app. I also think the example app should be upgraded at some point as well (this could be the test app we've talked about, for example)